src/sass/modules/sidebar.scss

```diff
@@ -29,6 +29,10 @@
   padding-top: 16px;
 }
 
+.sidebar__timestamps-created {
```

Comment:
Not much about this style is "created" related.  If we added two other timestamps below the two we have now what would be a better reusable class name?
Also "timestamps" and "created" are really pushing domain instead of generic.  I think `sidebar__timestamps` was a domain class renamed..  really this one and the one above should talk about more what this style is doing for the sidebar and not the content that it contains.

Reply:
`.sidebar__footer` and `.sidebar__footer-spacing`?

Reply:
or `sidebar__footer--left` ?

Reply:
🤘